src/main/java/TaskList.java

```diff
@@ -0,0 +1,62 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class TaskList {
+    private static ArrayList<Task> tasks = new ArrayList<>();
+    public static void listTasks() {
+        System.out.println("Here are the tasks in your list:");
+        for(int i = 0; i < tasks.size(); i += 1){
+            System.out.println((i + 1) + ". " + tasks.get(i).fullDescription());
+        }
+    }
+    public static void addToList(String command) {
```

Comment:
Perhaps it may be easier to see each function if you added a new line in between, but it is personal preference.